server/src/main/java/org/opensearch/search/backpressure/SearchBackpressureService.java

```diff
@@ -308,7 +308,7 @@ SearchBackpressureSettings getSettings() {
         return settings;
     }
 
-    SearchBackpressureState getSearchBackpressureStats(Class<? extends SearchBackpressureTask> taskType) {
+    SearchBackpressureState getSearchBackpressureState(Class<? extends SearchBackpressureTask> taskType) {
```

Comment:
Why do we need these refactoring? Do we plan to backport this again?

Reply:
We have already merged this backporting to 2.x. https://github.com/opensearch-project/OpenSearch/pull/6194